raco/language/myrialang.py

```diff
@@ -643,6 +656,9 @@ def __repr__(self):
         return "{op}({inp!r}, {hc!r})".format(op=self.opname(), inp=self.input,
                                               hc=self.hash_columns)
 
+    def partitioning(self):
```

Comment:
Shouldn't `partitioning` for the Producer side of an exchange operator reflect the partitioning it will see from its input (i.e., the actual partitioning of tuples on the worker where it's running)?